2023-12-15T00:24:49.5403627Z ##[group]Run actions/setup-python@v4
2023-12-15T00:24:49.5404226Z with:
2023-12-15T00:24:49.5404578Z   python-version: 3.8
2023-12-15T00:24:49.5404916Z   architecture: x64
2023-12-15T00:24:49.5405339Z   cache: pip
2023-12-15T00:24:49.5405720Z   check-latest: false
2023-12-15T00:24:49.5406294Z   token: ***
2023-12-15T00:24:49.5406606Z   update-environment: true
2023-12-15T00:24:49.5406988Z   allow-prereleases: false
2023-12-15T00:24:49.5407433Z ##[endgroup]
2023-12-15T00:24:49.6782464Z ##[group]Installed versions
2023-12-15T00:24:49.6856603Z Successfully set up CPython (3.8.18)
2023-12-15T00:24:49.6857884Z ##[endgroup]
2023-12-15T00:24:50.5938258Z [command]/opt/hostedtoolcache/Python/3.8.18/x64/bin/pip cache dir
2023-12-15T00:24:53.8828903Z /home/runner/.cache/pip
2023-12-15T00:24:54.2156749Z Received 6245959 of 6245959 (100.0%), 42.2 MBs/sec
2023-12-15T00:24:54.2157871Z Cache Size: ~6 MB (6245959 B)
2023-12-15T00:24:54.2187053Z [command]/usr/bin/tar -xf /home/runner/work/_temp/4d73a459-aa12-486d-bf6b-706a9d2c8a89/cache.tzst -P -C /home/runner/work/GitHub_Stats_A/GitHub_Stats_A --use-compress-program unzstd
2023-12-15T00:24:54.2461697Z Cache restored successfully
2023-12-15T00:24:54.2483129Z Cache restored from key: setup-python-Linux-22.04-Ubuntu-python-3.8.18-pip-60f2a6c0212a9b74a155cd81b58dd07fd8b180833132566260d95f4de42fe218
